staging/src/k8s.io/client-go/openapi/groupversion.go

```diff
@@ -39,7 +40,7 @@ func newGroupVersion(client *client, item handler3.OpenAPIV3DiscoveryGroupVersio
 	return &groupversion{client: client, item: item}
 }
 
-func (g *groupversion) Schema() (*openapi_v3.Document, error) {
```

Comment:
curious what changed? are we wanting a type other than *openapi_v3.Document now?

Reply:
I'm wondering if we should be returning *spec.Swagger instead of raw bytes... the byte accessor methods seem low-level for this client

Reply:
We already have a use-case though in kubectl where we want to decode into `map[string]interface{}`

Reply:
ok